code/src/sixsq/nuvla/server/resources/voucher.clj

```diff
@@ -0,0 +1,224 @@
+(ns sixsq.nuvla.server.resources.voucher
+  "
+This resource contains the structure for a voucher, which
+is to be issued by a third party and used by any Nuvla user.
+"
+  (:require
+    [sixsq.nuvla.auth.acl-resource :as a]
+    [sixsq.nuvla.auth.utils :as auth]
+    [sixsq.nuvla.db.impl :as db]
+    [sixsq.nuvla.server.resources.common.crud :as crud]
+    [sixsq.nuvla.server.resources.common.schema :as sc]
+    [sixsq.nuvla.server.resources.common.std-crud :as std-crud]
+    [sixsq.nuvla.server.resources.common.utils :as u]
+    [sixsq.nuvla.server.resources.resource-metadata :as md]
+    [sixsq.nuvla.server.resources.spec.voucher :as voucher]
+    [sixsq.nuvla.server.util.response :as r]
+    [clj-time.core :as time]
+    [sixsq.nuvla.server.util.metadata :as gen-md]
+    [clojure.pprint :as pp]))
```

Comment:
Unused.  Put imports in alphabetical order.  Can conveniently be done with `lein nsorg --replace`.

Reply:
done